workload/cfg_icni2_serving_resource_init.yml

```diff
@@ -107,6 +107,5 @@ jobs:
     jobIterationDelay: 0s
     jobPause: 1s
     objects:
-      - objectTemplate: {{if .BFD}} objectTemplates/pod_serving.yml {{else}} objectTemplates/pod_serving_no_bfd.yml {{end}}
```

Comment:
Sure you want to remove the if BFD condition?

Reply:
No, that was a mistake. Thanks for the catch.